dist/tools/functions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# run arguments as command, only print output when command errors
+quiet() {
+    OUT="$("$@" 2>&1)"
+    RES=$?
+    [ -n "$OUT" ] && [ $RES -ne 0 -o "$QUIET" = "0" ] && echo "$OUT"
```

Comment:
but wouldn't this mean that if the user supplies `QUIET=false` it does not have the correct effect?